classes/session/SessionManager.php

```diff
@@ -317,6 +223,171 @@ public static function hasSession(): bool
         // If the session isn't disabled and a cookie is present or a session was started in the current request
         return !SessionManager::isDisabled() && (isset($_COOKIE[Config::getVar('general', 'session_cookie_name')]) || !!session_id());
     }
+
+    private function configure(): void
+    {
+        $domain = $this->request->getServerHost(includePort: false);
+
+        // Configure PHP session parameters
+        ini_set('session.name', Config::getVar('general', 'session_cookie_name'));
+        ini_set('session.cookie_lifetime', 0);
+        ini_set('session.cookie_path', Config::getVar('general', 'session_cookie_path', $this->request->getBasePath() . '/'));
+        ini_set('session.cookie_domain', $domain);
+        ini_set('session.cookie_httponly', 1);
+        ini_set('session.cookie_samesite', Config::getVar('general', 'same_site', 'Lax'));
+        ini_set('session.cookie_secure', Config::getVar('security', 'force_ssl'));
+        ini_set('session.use_trans_sid', 0);
+        ini_set('session.serialize_handler', 'php');
+        ini_set('session.use_cookies', 1);
+        ini_set('session.gc_probability', 1);
+        ini_set('session.gc_maxlifetime', 60 * 60);
+        ini_set('session.cache_limiter', 'none');
+
+        session_set_save_handler($this, true);
```

Comment:
According to the manual, the second parameter has a default value of true, but I decided to ensure the `register_shutdown_function()` registration 👀